src/rtcmix/RTcmix.h

```diff
@@ -78,6 +78,8 @@ class RTcmix {
 
 	static bool interactive() { return rtInteractive; }
     static void setInteractive(bool interactive) { rtInteractive = interactive; }
+    static bool OSC() { return rtOSC; }
```

Comment:
Not sure what happened to my comments that were attached to these.  I would like the boolean setters' and getters' names to better indicate what they are doing - so, perhaps setOSCEnabled()? OSCIsEnabled()?